lms/resources/lti_launch.py

```diff
@@ -14,15 +14,6 @@
 class LTILaunchResource:
     """Context resource for LTI launch requests."""
 
-    DISPLAY_NAME_MAX_LENGTH = 30
-    """The maximum length of an h display name."""
-
-    GROUP_NAME_MAX_LENGTH = 25
-    """The maximum length of an h group name."""
-
-    USERNAME_MAX_LENGTH = 30
-    """The maximum length of an h username."""
```

Comment:
Each of these is only used in one method, so I've just inlined them into the methods where they're used.

Reply:
:+1: Even more localism. These can travel with that function if it ever moves. On the flip side, if we did add a colaborator class that's only doing this, I think these are fine top level constants.